release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2682,3 +2682,28 @@ link:https://access.redhat.com/solutions/6828761[{product-title} 4.9.25 containe
 ==== Updating
 
 To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-26"]
+=== RHBA-2022:0861 - {product-title} 4.9.26 bug fix and security update
+
+Issued: 2022-03-29
+
+{product-title} release 4.9.26 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:1022[RHBA-2022:1022] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:1021[RHSA-2022:1021] advisory.
```

Comment:
These advisory numbers (1022) don't match the ones above, at the title of this section (0861).

Reply:
Thanks for the catch! Advisory number updated!